canonical/canonical.com
# fix: apply fix for search being submit-able

## Done

- apply fix for search being submit-able 

### Flyby work

(I have added comments pointing out the changes as there are a lot of files)
- Format js
- Add `js-submit-button` where is was missing to enable the loading spinner
- Add functionality so that if you click outside the modal it closes 

## QA

- Open [demo page with a form](https://canonical-com-1431.demos.haus/solutions/ai)
- Open the the modal, click inside the modal and see nothing happens. Click outside the modal and see that it closes.
- Open the modal, fill it in and submit it. See that the submit button is disabled and have the [vanilla loading spinner](https://vanillaframework.io/docs/patterns/icons)
- Check this for a few of the different forms this change was applied to

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-16074
